apertium-kmr-eng.kmr-eng.lrx

```diff
@@ -142,5 +144,93 @@
     <match lemma="keç" tags="n.*"><select lemma="daughter" tags="n.*"/></match>
     <match tags="prn.pers.*"/>
   </rule>
-</rules>
 
+  <rule weight="1.1">
+    <match lemma="za" tags="n.*"><select lemma="son" tags="n.*"/>
+    </match>
+    <match tags="prn.pers.*"/>
+  </rule>
+
+  <rule weight="1.1">
+	  <match lemma="bi" tags="pr.*">
+	    <select lemma="in" tags="pr.*"/>
```

Comment:
When might it get translated as "with"?

Reply:
From the articles that I analyzed (Wikipedia), when a language was preceded by 'bi,' it always meant "in [language]."  For example: "Zimanê inglîzî (bi inglîzî: English)" [English language (in English: English)].

Reply:
I can't think of a case where with would be more appropriate than in, when it's regarding a language.

Reply:
I'm asking about a case when "with" would be more appropriate in general, not as something related to your rules.  I.e., when do we want to the other translation selected?  Usually in `lrx` you need to define a default case to ensure that you don't always get the output of your more specific rule.  That is, you can't be sure which translation `dix` is going to output left to its own devices, so in a situation like this you might just always get "in" and never with.

Reply:
I've added a rule to deal with that.

Reply:
"in" for languages and "with" otherwise is a good rule for _bi_, we can approve this.

Reply:
What does it in cases that aren't languages or pronouns?

Reply:
Looks like it's usually either "at" or "be."  Should I add a rule so that _bi_ is "with" unless specified otherwise?

Reply:
Yeah, a point I've been trying to make for some time is that you should probably do this for all the words you added....